pkg/combustion/kubernetes.go

```diff
@@ -57,6 +58,25 @@ func configureKubernetes(ctx *image.Context) ([]string, error) {
 		return nil, fmt.Errorf("configuring kubernetes components: %w", err)
 	}
 
+	manifestURLs := ctx.ImageDefinition.Kubernetes.Manifests.URLs
+	localManifestsComponentDir := filepath.Join("kubernetes", "manifests")
+	localManifestsConfigured := isComponentConfigured(ctx, localManifestsComponentDir)
+	if localManifestsConfigured || len(manifestURLs) != 0 {
+		localManifestsSrcDir := filepath.Join(ctx.ImageConfigDir, "kubernetes", "manifests")
+		k8sCombustionDir := filepath.Join(ctx.CombustionDir, k8sDir)
+		err = os.MkdirAll(k8sCombustionDir, os.ModePerm)
```

Comment:
This is something the artefact downloader is already doing. I'd suggest to just use another directory on the root combustion level until I sort out the kubernetes/ structure.

Reply:
Will update 👍